css/main.css

```
@@ -0,0 +1,114 @@
+.top-bottom-section {
+  height: 540px;
+  background: #cfe5ff;
+}
+
+.main-section {
+  background-color: #fcfcfc;
+  overflow: hidden;
+  border-radius: 12px;
+  max-height: 444px;
+  max-width: 988px;
+  margin: 0 auto 276px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.main-section img {
+  min-width: 430px;
+}
+
+.part-1 {
+  display: flex;
+  align-items: end;
+  justify-content: center;
+  margin-bottom: 138px;
+}
+
+.part-1 > div:first-child {
+  margin-bottom: 100px;
+  margin-right: 7px;
+  width: 367px;
+  min-width: 140px;
+}
+
+.part-1 h2 {
+  margin-bottom: 32px;
+}
+
+.part-1 a {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 56px;
+  border-radius: 40px;
+  background-color: #3692ff;
+  font-size: 1.5rem;
+  font-weight: 600;
+  line-height: 2rem;
+  letter-spacing: 0%;
+  color: #f9fafb;
+}
+
+.part-2 {
+  padding: 0 23.5px;
+}
+
+.part-2 > div:first-child {
+  margin-right: 64px;
+}
+
+.part-3 {
+  padding: 0 13px;
+}
+
+.part-3 h2,
+.part-3 p,
+.part-3 div {
+  text-align: right;
+}
+
+.part-3 > div:first-child {
+  width: 317px;
+  margin-right: 64px;
+}
+
+.part-4 > div:first-child {
+  margin-right: 64px;
+}
+
+.part-5 {
+  display: flex;
+  align-items: end;
+  justify-content: center;
+}
+
+.part-5 > div:first-child {
+  margin-bottom: 172.5px;
+}
+
+.highlight {
+  color: #3692ff;
+  font-size: 1.2rem;
+  font-weight: 800;
+  line-height: 1.5rem;
+  letter-spacing: 0%;
+  vertical-align: middle;
+}
+
+.sub-title {
+  font-weight: 700;
+  font-size: 2.5rem;
+  line-height: 140%;
+  vertical-align: middle;
+}
+
+.description {
+  font-weight: 500;
+  font-style: Medium;
+  font-size: 1.5rem;
+  line-height: 2rem;
+  letter-spacing: 0%;
```

❗️ 수정요청
letter-spacing 속성은 px단위로 입력하셔야 합니다. 0을 주고 싶으시다면 0 혹은 0px로 입력해주세요.

https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing


```suggestion
  letter-spacing: 0; // 단위가 필요없는 숫자이니 0을 추천드립니다!
```